packages/core/src/common/uri.ts

```diff
@@ -18,16 +18,17 @@ import { URI as Uri } from 'vscode-uri';
 import { Path } from './path';
 
 export default class URI {
-
     private readonly codeUri: Uri;
     private _path: Path | undefined;
+    private _contents: string | undefined;
```

Comment:
why do we need this? please revert

Reply:
@akosyakov 
cc: @vince-fugnitto @amiramw 

1. Thanks for review.
2. This was done after @RomanNikitenko found performance issue - which doesn't really relate to this PR (as you can reproduce it using search & replace) - but I thought I would try to solve it. 

The issue this want to address is this: 
When doing search &replace for instance, we are using URI withQuery to save file contents in URI query 

![image](https://user-images.githubusercontent.com/71069170/99366259-09042d80-28c1-11eb-9b4d-9e98cd51ac58.png)

The larger the file is - the longer it takes to display it - for instance opening this search/replace file took ~5-6 seconds and that is mainly because of "percentDecode" function which decodes entire uri querystring.
![image](https://user-images.githubusercontent.com/71069170/99366330-233e0b80-28c1-11eb-82c6-88285fcf8c00.png)

Please advise
Thanks

Reply:
@akosyakov @vince-fugnitto 
cc: @amiramw 
Reverted this changes - removed "contents" from uri.